localization/english/wc_regions_l_english.yml

```diff
@@ -0,0 +1,9 @@
+﻿l_english:
+ world_innovation_elephants:0 "Veiled Isles"
+ world_innovation_camel:0 "Tanaris or Uldum"
```

Comment:
It's localized twice. Plus, `world_innovation_camel` is wrong name. Also, don't forget to change other languages.